packages/local_auth/lib/local_auth.dart

```diff
@@ -72,4 +72,13 @@ class LocalAuthentication {
     }
     return await _channel.invokeMethod('authenticateWithBiometrics', args);
   }
+
+  /// Returns biometry type if enrolled on the device
+  ///
+  /// Returns a [Future] String with the following possibilities:
+  /// - iOS: 'faceID', 'touchID', 'notEnrolled' or 'notAvailable'
+  /// - Android: 'fingerprint', 'notEnrolled' or 'notAvailable'
+  Future<String> getBiometryType() async {
```

Comment:
Map the returned string to an enum instead so that it is easier for applications to use your API (also see the suggested API in my first comment).